prometheus-to-sd/Dockerfile

```diff
@@ -12,10 +12,20 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM gcr.io/google-containers/debian-base-amd64:0.3
+FROM golang:1.9-alpine as builder
+ARG DEP_VERSION="0.4.1"
+RUN apk update && apk add bash git
+ADD https://github.com/golang/dep/releases/download/v${DEP_VERSION}/dep-linux-amd64 /usr/bin/dep
+RUN chmod +x /usr/bin/dep
+WORKDIR ${GOPATH}/src/github.com/GoogleCloudPlatform/k8s-stackdriver/prometheus-to-sd
+COPY Gopkg.toml Gopkg.lock ./
+RUN dep ensure -vendor-only
+COPY . ./
+RUN CGO_ENABLED=0 GOOS=linux go build -o /usr/bin/monitor
 
+FROM gcr.io/google-containers/debian-base-amd64:0.3
 RUN clean-install ca-certificates
-
-#   nobody:nobody
 USER 65534:65534
-COPY monitor /
+COPY --from=builder /usr/bin/monitor /usr/bin/
```

Comment:
Please don't change binary location

Reply:
why?

Reply:
Because we are depending on it in kubernetes manifests. It's a breaking change that is not needed.